ts/components/portfolio/OperationsComponent.tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  Body,
+  Grid,
+  Icon,
+  Left,
+  List,
+  ListItem,
+  Right,
+  Row,
+  Text
+} from "native-base";
+import * as React from "react";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+
+import { PortfolioAPI } from "../../api/portfolio/portfolio-api";
+import I18n from "../../i18n";
+import ROUTES from "../../navigation/routes";
+import { CreditCard } from "../../types/portfolio/CreditCard";
+import { Operation } from "../../types/portfolio/types";
+import { PortfolioStyles } from "../styles";
+
+type Props = {
+  parent: string,
+  title: string,
+  totalAmount: string,
+  navigation: NavigationScreenProp<NavigationState>,
+  operations: ReadonlyArray<Operation>
+}
+
+type State = {
+  data: ReadonlyArray<Operation>
+}
+
+/**
+ * Operations" List component
+ */
+export class OperationsList extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props)
+    this.state = { data: Array.from(props.operations) }
+  }
+
+  public render(): React.ReactNode {
+    const { navigate } = this.props.navigation
+    const ops = this.state.data
+
+    if (ops === null || ops === undefined) {
```

Comment:
from the type of `State`, this case never happens (`state.data` is always defined)